### DISCUSSION TITLE
tailwindcss @apply triggers endless .eslintcache updates

### DISCUSSION THREAD

**Discussion by root5427** [nuxt/nuxt#10235] [Questions (Nuxt 2)] [answered]
### Versions:
- "nuxt": "^2.15.8",
- "webpack": "^4.46.0"
-  "@nuxt/postcss8": "^1.1.3"
-  "@nuxtjs/eslint-config": "^8.0.0"
- "@nuxtjs/eslint-module": "^3.0.2"
- "autoprefixer": "latest"
- "postcss": "latest"
- "tailwindcss": "^3.0.16"

---

### Issue
Using `@apply` in <style> (with scss scope) causes .eslintcache to be updated indefinitely.
Replacing `@apply` with the appropriate css rule no longer causes the problem.

Can anyone help with this? Should I use tailwindcss v2 to use `@apply` without any problem? Or is there any solution for this?

**Comment by dsk90it** [comment]
@root5427 Same issue for my new nuxt 2 + [tailwind 3 setup](https://tailwindcss.com/docs/guides/nuxtjs). After turning off eslint cache in nuxt.config.js `eslint: { cache: false, },` works fine 🎉.  But need a better solution for this.

**Reply by root5427** [reply]
Yes. I came with this solution too. But, as you said, we need a better solution for this

**Comment by cannap** [comment]
same here

**Comment by Errichamonda** [comment]
Not sure if it's tailwindcss fault but I have the same problem with .git\FETCH_HEAD

https://imgur.com/pde3INF

**Reply by cannap** [reply]
i have this also but i am not sure why

**Comment by cannap** [comment, marked as answer]
maybe this is a fix for this issue https://github.com/tailwindlabs/tailwindcss/pull/7524
